.travis.yml

```diff
@@ -8,6 +8,7 @@ branches:
     - trying
     - /^[\d.]+$/
     - /.+-stable$/
+    - /^bug\//
```

Comment:
adding this shouldn't be necessary, since you're making a PR into a branch that's already tested

Reply:
Fork. Fork. Fork.  Nothing is nicer than working on a PR in one's fork and discovering that you can't run a test on it because the branch name isn't allowed.

Also, I think it's also used in another popular repo, one that Nobu and some others here occasionally contribute to...

Reply:
Yes, the last commit is unnecessary, and I'll remove it.
It is just to run CIs before creating this PR.